gemini/agent-engine/evaluating_crewai_agent_engine_customized_template.ipynb

```
@@ -0,0 +1,1696 @@
+        "\n",
+        "LOCATION = os.environ.get(\"GOOGLE_CLOUD_REGION\", \"us-central1\")\n",
+        "\n",
+        "BUCKET_NAME = \"[your-bucket-name]\"  # @param {type: \"string\", placeholder: \"[your-bucket-name]\", isTemplate: true}\n",
+        "\n",
+        "if not BUCKET_NAME or BUCKET_NAME == \"[your-bucket-name]\":\n",
+        "    BUCKET_NAME = f\"{PROJECT_ID}-bucket\"\n",
+        "\n",
+        "BUCKET_URI = f\"gs://{BUCKET_NAME}\"\n",
+        "\n",
+        "! gsutil mb -p $PROJECT_ID -l $LOCATION $BUCKET_URI\n",
+        "\n",
```

Ruff (0.12.2)

Unnecessary lookup of dictionary value by key

Use existing variable

(PLR1733)